### ISSUE TITLE
Missing lib64

### ISSUE THREAD

**Issue by martinb3** [rackerlabs/lambda-uploader#25]
According to [this tutorial from AWS](http://docs.aws.amazon.com/lambda/latest/dg/python-walkthrough-s3-events-adminuser-create-test-function-create-function.html), we're missing the lib64 directory from being included in packages:
```
Add the contents of lib and lib64 site-packages to your .zip file.
$ cd  $VIRTUAL_ENV/lib/python2.7/site-packages
$ zip -r9 ~/lambda_function.zip * 
$ cd $VIRTUAL_ENV/lib64/python2.7/site-packages
$ zip -r9 ~/lambda_function.zip *
```

We should probably be including `lib64/python2.7/site-packages` in the uploaded lambda function package, in addition to what we're including now.

**Comment by jarosser06** [comment]
PR #21 fixes this but it needs some cleanup.  Also Before copying the lib64 libraries there needs to be a check to see if lib64 is a symlink before that happens though.  In my limited testing so far on Linux the lib64 directory has always been a symlink.

ls -lh
total 16K
drwxrwxr-x. 2 jim jim 4.0K Nov 17 20:22 bin
drwxrwxr-x. 2 jim jim 4.0K Nov 17 20:22 include
drwxrwxr-x. 3 jim jim 4.0K Nov 17 20:22 lib
lrwxrwxrwx. 1 jim jim    3 Nov 17 20:22 lib64 -> lib
-rw-rw-r--. 1 jim jim   60 Nov 17 20:22 pip-selfcheck.json

**Comment by martinb3** [comment]
The one I built on Amazon Linux wasn't a symlink:
```
[root@ip-172-31-63-128 ~]# ls -la amazonlinux/
total 32
drwxr-xr-x 7 root root 4096 Nov 17 22:01 .
dr-xr-x--- 7 root root 4096 Nov 17 22:05 ..
drwxr-xr-x 2 root root 4096 Nov 17 22:01 bin
drwxr-xr-x 2 root root 4096 Nov 17 22:01 include
drwxr-xr-x 3 root root 4096 Nov 17 22:01 lib
drwxr-xr-x 3 root root 4096 Nov 17 22:01 lib64
drwxr-xr-x 2 root root 4096 Nov 17 22:01 local
-rw-r--r-- 1 root root   60 Nov 17 22:01 pip-selfcheck.json
```

**Comment by jarosser06** [comment]
Should be fixed in PR #28

**Comment by martinb3** [comment]
Fixed in #28.
